sklearn/utils/__init__.py

```diff
@@ -498,6 +498,19 @@ def indices_to_mask(indices, mask_length):
     -------
     mask : 1d boolean nd-array
         Boolean array that is True where indices are present, else False.
+
+    Examples
+    --------
+    >>> from sklearn.utils import indices_to_mask
+    >>> indices = [1, 2 , 3, 4]
+    >>> indices_to_mask(indices, 5)
+    array([False,  True,  True,  True,  True], dtype=bool)
+    >>> indices_to_mask(indices, 4)
+    Traceback (most recent call last):
+      File "<stdin>", line 1, in <module>
+      File "sklearn/utils/__init__.py", line 503, in indices_to_mask
```

Comment:
This is too brittle a test. Do we really need an example of an error here?

Reply:
I got inspired from this: https://github.com/scikit-learn/scikit-learn/blob/master/sklearn/utils/__init__.py#L457-L469. Also, while preparing the example, It wasn't obvious from the parameter description that the **mask_length must be greater than max(indices)**. So I thought making this obvious with an example would help the user.

Reply:
But I think, adding a clean example is better. The ambiguity in the parameter **mask_length** can be resolved later by enhancing the description of the parameter itself.